README.md

```diff
@@ -1,18 +1,29 @@
 # vscode-flowr
 flowR Extension for Visual Studio Code 
 
-## Features
-TO DO
+## Installing
 
-## Requirements
-TO DO
+### From Visual Studio Marketplace
+vscode-flowr is not available on the Visual Studio Marketplace yet. For now, we recommend installing from the artifact, or building from source.
 
-## Extension Settings
-TO DO
+### From GitHub Releases
+There are no official releases yet. Stay tuned!
 
-# Developing
+### From build artifacts
```

Comment:
Note that I Don't Use Titlecase for these headers. The two headers above only appear that way because they use proper nouns!